### DISCUSSION TITLE
Syntax highlighting in VS Code

### DISCUSSION THREAD

**Discussion by rkh** [fluent-vue/fluent-vue#853] [Q&A] [answered]
Has anyone figured out how to get syntax highlighting to work in VS Code (or another editor) for `SFCFluentPlugin` / unplugin-fluent-vue?

**Comment by Demivan** [comment]
Hi.
For JetBrains IDEs there is a plugin already: https://plugins.jetbrains.com/plugin/18416-fluent-language.
There is no VS Code extension yet. It is not hard to implement syntax highlighting extension for VS Code. I will work on it. Should be up in few days.

**Reply by Demivan** [reply, marked as answer]
VS Code extension published to Microsoft Marketplace (https://marketplace.visualstudio.com/items?itemName=fluent-vue.fluent-vue) and to Open VSX Registry (https://open-vsx.org/extension/fluent-vue/fluent-vue) (namespace not verified yet)

@rkh Please, check if it is working as expected and give feedback on it.

**Reply by rkh** [reply]
Oh amazing, that was fast. I can confirm it works.
